homeassistant/components/sensor/xiaomi_aqara.py

```diff
@@ -7,6 +7,14 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+SENSOR_TYPES = {
+    'temperature': [TEMP_CELSIUS, 'thermometer'],
```

Comment:
All need to be prefixed with `mdi:`

Reply:
Thanks, forgot this when I changed my mind on how to do this :+1: